etc/conf/atTelemetry_salobj.yaml

```diff
@@ -0,0 +1,271 @@
+# Configuration file for the AuxTelescope
+
+# Default logging level
+loglevel: 'INFO'
+
+# Path to the log location and other log definitions
+logfile: 'www/HeaderService.log'
+log_format: '[%(asctime)s.%(msecs)03d][%(levelname)s][%(name)s][%(funcName)s] %(message)s'
+log_format_date: '%Y-%m-%d %H:%M:%S'
+
+# The camera vendor
+vendor: ITL
+
+# The ts_xml version we are using -- for now this is just curated and need to be changed manually when we change version
+ts_xml: "3.10.0"
+
+# port_number for webserver
+port_number: 8000
+
+# The URL format to be used
+url_format: "http://{ip_address}:{port_number}/{filename_HDR}"
+
+# Default write_mode (fits or string)
+write_mode: fits
+
+# Timeout for waiting for end of Telemetry event (in seconds)
+timeout_endTelem: 120
+
+# Default string delimeter (string mode only)
+hdu_delimiter: END
+
+# The file naming event source and definitions
+filepath: 'www/DMHS_filerepo'
+format_HDR: "{}.header"
+format_FITS: "{}.fits"
+imageName_event:
+    device: ATCamera
+    topic: startIntegration
+    value: imageName
+
+# The Events that we use to define where we collect
+start_collection_event:
+    device: ATCamera
+    topic: startIntegration
+
+end_collection_event:
+    device: ATCamera
+    topic: endOfImageTelemetry
+
+# The event for the image readout paramters
+imageParam_event:
+    device: ATCamera
+    topic: imageReadoutParameters
+
+# Telemetry values for images meta-data
+# https://confluence.lsstcorp.org/pages/viewpage.action?spaceKey=SYSENG&title=Auxiliary+Telescope+Header+Information+Topic+Mapping
+telemetry:
+
+ DATE-OBS:
+  device: ATCamera
+  Stype: Event
+  topic: startIntegration
+  value: timeStamp
+  collect_after_event: start_collection_event
+
+ DATE-BEG:
+  device: ATCamera
+  Stype: Event
+  topic: startIntegration
+  value: timeStamp
+  collect_after_event: start_collection_event
+
+ DATE-END:
+  device: ATCamera
+  Stype: Event
+  topic: endReadout
+  value: timeStamp
+  collect_after_event: end_collection_event
+
+ OBSID:
+  device: ATCamera
+  Stype: Event
+  topic: startIntegration
+  value: imageName
+  collect_after_event: start_collection_event
+
+ SEQNUM:
+  device: ATCamera
+  Stype: Event
+  topic: startIntegration
+  value: imageIndex
+  collect_after_event: start_collection_event
+
+ GROUPID:
+  device: ATCamera
+  Stype: Event
+  topic: startIntegration
+  value: imageSequenceName
+  collect_after_event: start_collection_event
+
+# ----------------------------------------
+# -- still not available from SAL --------
+# OBSTYPE:
```

Comment:
I assume making this available is work that is planned?

If there is no Jira ticket to add the feature, might you create one? If there is such a ticket then please file a ticket against HeaderService to update this code once the other ticket is implemented (and make it block against the other).

Reply:
Yes, there is a jira ticket. Waiting for Tony to implement this. Hopefully in the next xml version.